Doc/tools/susp-ignored.csv

```diff
@@ -180,15 +180,15 @@ library/pickle,,:memory,"conn = sqlite3.connect("":memory:"")"
 library/posix,,`,"CFLAGS=""`getconf LFS_CFLAGS`"" OPT=""-g -O2 $CFLAGS"""
 library/pprint,,::,"'Programming Language :: Python :: 2.6',"
 library/pprint,,::,"'Programming Language :: Python :: 2.7',"
-library/pprint,225,::,"'classifiers': ['Development Status :: 3 - Alpha',"
-library/pprint,225,::,"'Intended Audience :: Developers',"
-library/pprint,225,::,"'License :: OSI Approved :: MIT License',"
-library/pprint,225,::,"'Programming Language :: Python :: 2',"
-library/pprint,225,::,"'Programming Language :: Python :: 3',"
-library/pprint,225,::,"'Programming Language :: Python :: 3.2',"
-library/pprint,225,::,"'Programming Language :: Python :: 3.3',"
-library/pprint,225,::,"'Programming Language :: Python :: 3.4',"
-library/pprint,225,::,"'Topic :: Software Development :: Build Tools'],"
+library/pprint,252,::,"'classifiers': ['Development Status :: 3 - Alpha',"
+library/pprint,252,::,"'Intended Audience :: Developers',"
+library/pprint,252,::,"'License :: OSI Approved :: MIT License',"
+library/pprint,252,::,"'Programming Language :: Python :: 2',"
+library/pprint,252,::,"'Programming Language :: Python :: 3',"
+library/pprint,252,::,"'Programming Language :: Python :: 3.2',"
+library/pprint,252,::,"'Programming Language :: Python :: 3.3',"
+library/pprint,252,::,"'Programming Language :: Python :: 3.4',"
+library/pprint,252,::,"'Topic :: Software Development :: Build Tools'],"
```

Comment:
Sorry, a question, why did you make this changes?

Reply:
The CI runs https://github.com/python/cpython/blob/master/Doc/tools/extensions/suspicious.py which looks for suspicious constructs in the documentation, you can use `Doc/tools/susp-ignored.csv` to whitelist lines that are correct. Here I removed some lines in `pprint.rst` which made the CI complains about `Programming Language :: Python :: 3.2` and others.

I needed to update the whitelist as those are correct.

You can see the CI fail for `bb2c055` at https://travis-ci.org/python/cpython/jobs/489545059#L556.

Reply:
It should be okay to remove the line numbers; look at the rest of the file